src/vcSceneLayer.cpp

```diff
@@ -596,10 +596,13 @@ udResult vcSceneLayer_LoadTextureData(const vcSceneLayer *pSceneLayer, vcSceneLa
         continue;
```

Comment:
pPathBuffer doesn't get allocated, can udSprintf work like this?

Reply:
`pPathBuffer` is allocated via `udSprintf`, when you pass in a `char **` as the first parameter it will allocate a buffer big enough for you.